### PR TITLE
Test patch calls

### LINKED ISSUE #1682: Find a way to test PATCH requests
The current implementation returns a 599 error, try to find a way of testing this.

### PR DESCRIPTION
**Pull request by squirrelo**
This fixes #1682 to properly make patch calls for tests.

data is now required, as without it the following is thrown and returned as part of the request:
ValueError('Body must not be None for method PATCH (unless allow_nonstandard_methods is true)

### CODE REVIEW

File: qiita_pet/test/test_prep_template.py [hunk 1]
```diff
@@ -42,14 +42,13 @@ def test_post(self):
...
-        # exp = {'status': 'success', 'message': ''}
-        # self.assertEqual(laods(response.body), exp)
+        arguments = {'op': 'replace',
+                     'path': '/1/investigation_type/',
+                     'value': 'Cancer Genomics'}
+        response = self.patch('/prep_template/', data=arguments)
+        self.assertEqual(response.code, 200)
+        exp = {'status': 'success', 'message': ''}
+        self.assertEqual(loads(response.body), exp)
         pass
```
Comment (josenavas) [inline qiita_pet/test/test_prep_template.py]: Remove the pass

**Comment by josenavas** [comment]
1 small comment 👍

**Comment by squirrelo** [comment]
Comment addressed, ready to go.

**Comment by antgonza** [comment]
@squirrelo, could you confirm that your changes are OK with the new portal changes? Thanks.

**Comment by squirrelo** [comment]
Shouldn't have any issues since this just touches test code, but pulled in master and restarted tests to make sure.

**Comment by antgonza** [comment]
Thanks, will merge once tests pass.

**Pull request merged**